### PR TITLE
Updates for the Control panel component README

### PR DESCRIPTION
**Pull request by sarahmcdonald7**
This PR includes updates for the Control panel readme(added a description/screenshot) and updated descriptions for the 2 properties in the control panel .tsx file.

### CODE REVIEW

File: packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx [hunk 1]
```diff
@@ -19,10 +19,16 @@ export class InstantAppsControlPanel {
...
+  
+  /**
+   * Determine which widgets or components to display in the control panel
+   */
   @Prop()
   components: ControlPanelComponent[] = [];
-
+ 
+  /**
+   * MapView or SceneView
```
Comment (rslibed) [inline packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx:30]: Suggested edit: A reference to the MapView or SceneView.

Comment (rslibed) [reply packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx:30]: This similar to how the ArcGIS Maps SDK for JS docs the `view` property

**Review by rslibed — APPROVED** [review]
@sarahmcdonald7 Looks good! Added a comment

File: packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx (hunk 1 above)
Comment (sarahmcdonald7) [reply packages/instant-apps-components/src/components/instant-apps-control-panel/instant-apps-control-panel.tsx:30]: Sounds good, I will update that and include it when I update more of the md files. Thanks!

**Comment by rslibed** [comment]
feel free to merge this whenever you're ready

**Pull request merged**